plugins/modules/zos_fetch.py

```diff
@@ -170,66 +167,66 @@
 
 RETURN = r"""
 file:
-    description: The source file path on remote machine
+    description: The source file path or data set on the remote machine.
```

Comment:
@asifmahmud - is this returning the value used as `src` , ie a USS file or Data Set ?

Reply:
@ddimatos - Yes